README.md

```diff
@@ -1 +1,31 @@
-# flexy-thingworx-connector
\ No newline at end of file
+# Flexy Thingworx Connector
+
+The Flexy Thingworx Connector provides a connector-based solution development kit for linking the Ewon Flexy to a Thingworx instance.
+
+There are two components that make up the Flexy Thingworx Connector, a Thingworx service and a Flexy Java app.
+
+## Table of Contents
+1. [Thingworx Service Component](#thingworx-service-component)
+2. [Flexy Java App Component](#flexy-java-app-component)
+   1. [Installation](#installation)
+   2. [Source Code](#source-code)
+      1. [Development Environment](#development-environment)
+
+## Thingworx Service Component
+The Thingworx service component must be installed for both data path configurations (Direct and Talk2M).
+
+Information and instructions for the Thingworx service component are available in a separate document, [thingworx-import/README.md](thingworx-import/README.md).
+
+## Flexy Java App Component
```

Comment:
I think its best for clarity to specify application instead of app. App is really associated with phone/mobile apps.

Reply:
fixed